### PR TITLE
chore(ux): apply followup ux feedback

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR applies more general cleanup and fine tuning of the UI.

Screen Shot 2019-06-04 at 9 57 27 AM

remove unnecessary toolbar group
use kebab toggle
add test ids
replace missing test for header menus
automatically close the sidebar menu on small screen sizes
add skip to content link for accessibility

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2131/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/packages/ui/src/Layout/AppLayout.tsx [hunk 1]
```diff
@@ -66,14 +67,28 @@ export const AppLayout: React.FunctionComponent<ILayoutBase> = ({
...
+    <SkipToContent href="#main-content-page-layout-default-nav">
+      Skip to Content
+    </SkipToContent>
+  );
+  const [isMobileView, setIsMobileView] = React.useState(false);
+  const [isNavOpenMobile, setIsNavOpenMobile] = React.useState(false);
+  const onNavToggleMobile = () => {
+    setIsNavOpenMobile(!isNavOpenMobile);
+  };
+  const onPageResize = (props: { mobileView: boolean; windowSize: number }) => {
```
Comment (riccardo-forina) [inline app/ui-react/packages/ui/src/Layout/AppLayout.tsx]: I do hope this is somehow debounced on the PF side... if not we'll need to debounce it here.

Comment (seanforyou23) [reply app/ui-react/packages/ui/src/Layout/AppLayout.tsx]: Yep - https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Page/Page.js#L57

Comment (riccardo-forina) [reply app/ui-react/packages/ui/src/Layout/AppLayout.tsx]: Cool thanks!

**Pull request merged**